testdata/fixtures/Json/test.json

```diff
@@ -21,7 +21,7 @@
   "arrayUint": [1231232, "0x12C980", "1231232"],
   "stringInt": "-12",
   "numberInt": -12,
-  "hexInt": "0x-C",
+  "hexInt": "-0xC",
```

Comment:
what's this?

Reply:
Signed hex integer—moved the sign

Reply:
I don't know why we even support signs on non-decimal representation, generally you would just use the twos complement to display it